Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -199,7 +199,13 @@ def pull_request_branch_point(cls, repository, args, **kwargs):
                     if remote in bp_remotes:
                         source_remote = remote
                         break
-            if source_remote and source_remote != 'origin':
+            if source_remote != repository.default_remote:
+                print("Making pull request against '{}' because that is where the branch point is from")
+                if branch_point.branch in repository.DEFAULT_BRANCHES:
+                    sys.stderr.write('Branch point is on the default branch\n')
```

Comment:
Nit: why sys.stderr.write instead of print here?

Reply:
Because this is a really weird state that indicates something wrong with your local setup, likely you checked out `main` from a non-origin remote.